docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+st2:
+  # st2api (9101),
+  # st2garbagecollector,
+  # st2stream (9102),
+  # st2actionrunner,
+  # st2auth (9100), 
+  # st2notifier,
+  # st2resultstracker,
+  # st2rulesengine,
+  # st2web (80)
+  image: st2:latest
+  expose:
+    - "22"
+    - "80"
+    - "443"
+    - "9100"
+    - "9101"
+    - "9102"
+  ports:
+    - "22:22"
+  links:
+    - mongo
+    - rmq
```

Comment:
Small request to avoid abbrs in favor of clarity.
Just obvious for users `rabbitmq` and `postgres` should work well.